tutorials/notebooks/color-excess/color-excess.ipynb

```diff
@@ -0,0 +1,866 @@
+    "$$\n",
+    "where $E_{B-V}$ is differential extinction $A_B-A_V$.  In this example, we show the $R_V$-parameterization for the Clayton, Cardelli, & Mathis (1989, CCM) and the Fitzpatrick (1999) models. [More model options are available in the `dust_extinction` documentation.](https://dust-extinction.readthedocs.io/en/latest/dust_extinction/model_flavors.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 358
+    },
+    "colab_type": "code",
+    "id": "b1uhXbRl79FR",
+    "outputId": "e8999675-8ac1-4398-deb2-3bc5d36342ae"
+   },
+   "outputs": [],
+   "source": [
+    "# Create wavelengths array.\n",
+    "wav = np.arange(0.1, 3.0, 0.001)*u.micron\n",
+    "\n",
+    "for model in [CCM89, F99]:\n",
+    "   for R in (2.0,3.0,4.0):\n",
+    "     # Initialize the extinction model\n",
+    "     ext = model(Rv=R)\n",
+    "     plt.plot(1/wav, ext(wav), label=model.name+' R='+str(R))\n",
+    "        \n",
+    "plt.xlabel('$\\lambda^{-1}$ ($\\mu$m$^{-1}$)')\n",
+    "plt.ylabel('A($\\lambda$) / A(V)')\n",
+    "plt.legend(loc='best')\n",
+    "plt.title('Some Extinction Laws')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "d-1XRyovX028"
+   },
+   "source": [
+    "Astronomers studying the ISM often display extinction curves against inverse wavelength (wavenumber) to show the ultraviolet variation, as we do here.  Infrared extinction varies much less and approaches zero at long wavelength in the absence of wavelength-independent, or grey, extinction."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "TMEWEKJf-iSL"
+   },
+   "source": [
+    "# Example 2: Deredden a Spectrum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "zTeDJ-dI1cQj"
+   },
+   "source": [
+    "Here we deredden (unextinguish) the IUE ultraviolet spectrum and optical photometry of the star $\\rho$ Oph (HD 147933).\n",
+    "\n",
+    "First, we will use astroquery to fetch the archival [IUE spectrum from MAST](https://archive.stsci.edu/iue/):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 193
+    },
+    "colab_type": "code",
+    "id": "Zdwr_mdZcDeh",
+    "outputId": "d274490e-e6e2-449f-aaed-29dd0908a308"
+   },
+   "outputs": [],
+   "source": [
+    "obsTable = Observations.query_object(\"HD 147933\",radius=\"1 arcsec\")\n",
+    "obsTable_spec=obsTable[obsTable['dataproduct_type']=='spectrum']\n",
+    "# obsTable_spec.pprint()\n",
+    "\n",
+    "obsids = ['3000022829']\n",
+    "dataProductsByID = Observations.get_product_list(obsids)\n",
+    "manifest = Observations.download_products(dataProductsByID)"
+   ]
```

Comment:
You might also want to add a line at the end here along the lines of:
```
assert np.all(manifest['Status']!='ERROR')
```
?  I was just testing it now and there was a mast problem but it was "hidden" because there's no check for a failed download here.

Reply:
Do we still need to do this?